### ISSUE TITLE
RFP - signal.portable

### ISSUE THREAD

**Issue by dtantono** [chocolatey-community/chocolatey-package-requests#1336]
Checklist

- [x] The package I am requesting does not already exist on https://chocolatey.org/packages;
- [x] There is no open issue for this package;
- [x] The issue title starts with 'RFP - ';
- [x] The download URL is public and not locked behind a paywall / login;

## Package Details

Software project URL :https://github.com/portapps/signal-portable
Direct download URL for the software / installer : https://github.com/portapps/signal-portable/releases/download/5.38.0-7/signal-portable-win64-5.38.0-7-setup.exe
Software summary / short description:
portable version of Signal

**Comment by mikeee** [comment]
The package hasn't been updated and isn't working at the moment. I'm happy to pick up this package and submit releases for it though.

**Comment by pauby** [comment]
The [`signal`](https://community.chocolatey.org/packages/signal) package is maintained by @majkinetor, and it would make sense for them to pick this up and maintain it too, if they want to. @mikeee would you be willing to reach out to @majkinetor through the [`signal`](https://community.chocolatey.org/packages/signal) package page and ask?

**Comment by Xav83** [comment]
Yes, indeed, I had trouble with the verification phase (cf. [package](https://community.chocolatey.org/packages/signal.portable/#versionhistory)). If you want, @mikeee, I can add you as a maintainer of the package so that you can push a new version of the package :wink:

**Comment by mikeee** [comment]
@Xav83 absolutely happy to take a look as well, my username is `mikeee`. If you don't mind I'll pull the skeleton from https://github.com/Xav83/chocolatey-packages/commit/204ea573aea365567d1988b850670b25173cb7cd

**Comment by Xav83** [comment]
@mikeee, I have added you as a maintainer of the [signal.portable chocolatey package](https://community.chocolatey.org/packages/signal.portable). And of course you can pull the skeleton of the https://github.com/Xav83/chocolatey-packages/commit/204ea573aea365567d1988b850670b25173cb7cd, I won't mind at all :wink:
